### ISSUE TITLE
avatar is misaligned with name

### ISSUE THREAD

**Issue by no-boot-device** [glitch-soc/mastodon#25]
It's just so ever slightly higher than it, for some reason

![image demonstrating the issue goes here](https://my.mixtape.moe/yhdezv.png)

* * * *

- [ ] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [ ] This bug happens on a [tagged release](https://github.com/tootsuite/mastodon/releases) and not on `master` (If you're a user, don't worry about this).

**Comment by marrus-sh** [comment]
I added additional padding around username/display names to make collapsed toots look nicer—this can be adjusted or removed though

**Comment by beatrix-bitrot** [comment]
Is this still a thing?

**Comment by marrus-sh** [comment]
I mean I haven't changed it so yes it is :P

Personally I don't mind it but others might. Here are some imgs for ref:

| dev.glitch.social | other websites |
| --- | --- |
| <img width="321" alt="image" src="https://user-images.githubusercontent.com/3212430/28221846-cbfb496a-6879-11e7-8a49-faed621dca24.png"> | <img width="316" alt="image" src="https://user-images.githubusercontent.com/3212430/28221865-e36ad2d2-6879-11e7-802a-1867d6935999.png"> |

You'll see that it's dropped just a lil on glitchsoc so that the header and the first line are centered in the avi instead of pinned to the top.

**Comment by MightyPork** [comment]
It's a riddle. It seems to be due to that `line-height: 24px` on the name box, but if you remove it, it's still off. And it was there because of the toggle icon, I think. 

I got it to look thus by removing the line height, adding some offset to the icons, bumping the image a little and adding y margin at the top, but I suspect I broke something else in the process

![screenshot_20170714_191247](https://user-images.githubusercontent.com/2041118/28222657-be994ffc-68c8-11e7-90be-b0ee5ff08bf2.png)
